frontend/packages/kubevirt-plugin/src/actions/provider.ts

```diff
@@ -1,8 +1,11 @@
 import * as React from 'react';
+import { Node } from '@patternfly/react-topology';
 import { CommonActionFactory } from '@console/app/src/actions/creators/common-factory';
 import { useK8sWatchResource } from '@console/internal/components/utils/k8s-watch-hook';
-import { referenceFor } from '@console/internal/module/k8s';
+import { referenceFor, modelFor } from '@console/internal/module/k8s';
 import { useK8sModel } from '@console/shared/src/hooks/useK8sModel';
+import { getModifyApplicationAction } from '@console/topology/src/actions';
```

Comment:
Why is kubevirt package importing stuff from topology?

Reply:
@rohitkrai03 Where do you recommend moving the `getModifyApplicationAction` action to ? this currently resides in the topology package

Reply:
Why do we need to contribute this action from kubevirt-plugin? isn't this a common action that comes with almost all workloads? This should be contributed from Topology package itself.